### PR TITLE
Add config for release please.

### PR DESCRIPTION
**Pull request by SilasKenneth**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/msgraph-sdk-php/pull/1532)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,12 @@
 
 ## Install the SDK
 You can install the PHP SDK with Composer by editing your `composer.json` file:
-```
+```json5
```
Comment (baywet) [inline README.md:10]:
```suggestion
```json
```

**Comment by SilasKenneth** [comment]
I would recommend using something other than 
```text
```json

```

to avoid the red highlighting for comments in json
Screenshot 2024-05-22 154041

--- 
That's the reason I used `json5`. We can also use `js`. What do you think @baywet?

**Comment by baywet** [comment]
@SilasKenneth have you tried jsonc (json with comments) ? I don't have a strong opinion either way. I didn't know json5 was a thing until your last message, sorry for creating confusion here. https://json-5.com/json-vs-json5

**Comment by SilasKenneth** [comment]
> @SilasKenneth have you tried jsonc (json with comments) ? I don't have a strong opinion either way. I didn't know json5 was a thing until your last message, sorry for creating confusion here. https://json-5.com/json-vs-json5

I just tried it now. Looks like it achieves the desired output too.

**Review by baywet — APPROVED** [review]
This is ready to merge, thanks for making the changes!

**Pull request merged**